benchmark/Naming.md

```diff
@@ -130,8 +130,8 @@ legacy-style).
 Prefer **`Char`** to `Character`, which can be combined with codepage or
 language prefix/suffix when necessary (`EmojiChar`, `ASCIIChar`). For benchmarks
 that measure `String`'s Unicode performance for various languages, use
-[two letter codes](https://en.wikipedia.org/wiki/ISO_639-1) instead of spelling
-out the whole language names.
+the [three letter codes](https://en.wikipedia.org/wiki/ISO_639-3) instead of
```

Comment:
These three-letter codes are pretty esoteric as compared to two-letter codes. I’ve pretty much never seen them used. What’s the thinking behind this change?

Reply:
From commit description:
> Use <del>for</del> ISO 639-3 <ins>for</ins> language variants 
> Matched the part of the ISO standard used for language code to match the one used on 
> https://www.unicode.org/udhr/translations.html
> 
> (I plan to use the Universal Declaration of Human Rights as corpus for string testing.)

Reply:
The two-letter codes don’t correspond to countries for either Chinese or English, two of the most common languages, so I doubt many people suffer from such a deception. I’m concerned that the three-letter codes, besides being much less common, have been critiqued as sometimes pejorative to the languages they’ve been assigned and that they are overseen by an opaque process internal to a missionary organization.

Reply:
🤦‍♂️ I'll be posting in the forums about string benchmarks soon, so you can share your concerns there.

Reply:
@xwu FYI: https://forums.swift.org/t/more-representative-string-benchmarks/18793